src/tests/UserSessionTest.test.ts

```diff
@@ -93,4 +93,26 @@ describe('user session tests', () => {
 
     expect(getSessionsResponse.sessions).toHaveLength(2);
   });
+
+  test('refresh token', async () => {
+    const authController = ControllerFactory.auth(conn);
+    const user = UserFactory.fake();
+    const session = UserSessionFactory.fake();
+    session.user = user;
+    session.userId = user.id;
+
+    await new DataFactory()
+      .createUsers(user)
+      .createUserSessions(session)
+      .write();
+    
+    const refreshToken = session.refreshToken;
+    const accessToken = session.accessToken;
+
+    const getSessionResponse = await authController.refreshToken(refreshToken);
+
+    expect(getSessionResponse.session.accessToken).not.toEqual(accessToken);
+    expect(getSessionResponse.session.refreshToken).not.toEqual(refreshToken);
+    expect(getSessionResponse.session.expiresAt).toBeCloseTo(Math.floor(new Date().getTime()) + 1000 * 60 * 60 * 24 * 30, -3);
```

Comment:
what is this expectation checking? that the expiration time is a month from now? and the -3? we should have the constant multiplication stored somewhere as an easy to read constant

Reply:
will address this in a separate PR